README.md

```diff
@@ -1,36 +1,43 @@
-# Quark
+# Cosmos SDK
 
-Quark is an [ABCI application](https://github.com/tendermint/abci) designed to
-be used with the [Tendermint consensus engine](https://tendermint.com/) to form
-a Proof-of-Stake cryptocurrency.  It also provides a general purpose framework
+![banner](cosmos-sdk-image.png)
+
+[![version](https://img.shields.io/github/tag/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/cosmos-sdk/releases/latest)
```

Comment:
this shows up as none because for some reason the releases didn't persist when the repo was transfered